modules/verifying-serverless-app-deployment.adoc

```diff
@@ -24,3 +31,29 @@ helloworld-go   http://helloworld-go.default.example.com   helloworld-go-4wsd2
 $ curl http://helloworld-go.default.example.com
 Hello World: Go Sample v1!
 ----
++
+----
+$ curl https://helloworld-go.default.example.com
+Hello World: Go Sample v1!
+----
+
+== Self signed certificates
+
+If you receive an error relating to a self signed certificate in the certificate chain, you can add the ``--insecure` flag to the curl command to ignore the error.
```

Comment:
Shall we add a note that this is not advised for production load and at most to test if things work?

Reply:
Why not.

Reply:
Because ignoring certificate validations allows for man-in-the-middle attacks. Hence the thing is called `--insecure`.

Reply:
Yeah. I said why not == let's do that. I guess it's common practice to put the warning there even though it assumes not very clever users.